### DISCUSSION TITLE
Vertical tabs hidden behind full screen

### DISCUSSION THREAD

**Discussion by Sheepyhead** [pulse-browser/browser#195] [Ideas]
This is the first browser I use with this new vertical tab setup, but as seen to the left in the attached screenshot (youtube video in full screen mode), the vertical tabs module is shown even when the browser is in full screen, unlike the normal horizontal tabs view. This seems like an oversight, but maybe it's just because the feature is still in development? Either way I wanted to bring attention to it, as ideally I'd like at least the option to have it be hidden when a Pulse window is in full screen.
![image](https://user-images.githubusercontent.com/15895891/211931432-d530f17f-a3d4-4d06-88ce-e7ac18bdaa37.png)

**Comment by PressJump** [comment]
Hey @Sheepyhead,
First of all, thanks for letting us know of the issue and we will make a branch to fix this in an upcoming release. We have fixed this kind of bug before on our sidebar but because vertical tabs were recently introduced it looks like it has been introduced to them as well like you have mentioned.

Tagging this as a bug.

**Comment by PressJump** [comment]
Made a issue for this discussion which is available here https://github.com/pulse-browser/browser/issues/196 for tracking purposes.
